client/client_test.go

```diff
@@ -39,18 +39,18 @@ func cleanUp() {
 	}
 }
 
-func NewJobMap() map[string]string {
+func NewJobMap() *job.Job {
 	scheduleTime := time.Now().Add(time.Minute * 5)
 	repeat := 1
 	delay := "P1DT10M10S"
 	parsedTime := scheduleTime.Format(time.RFC3339)
 	scheduleStr := fmt.Sprintf("R%d/%s/%s", repeat, parsedTime, delay)
```

Comment:
@ajvb Please put this exact logic in a helper somewhere!  This is valuable domain expertise you've acquired that I think Kala users can benefit from, especially while using Kala -- we should be able to create a `time.Time` object, pass it to a helper you define, then include that in an API call to Kala, all without needing to know the ins and outs of the time interval format you happened to choose.

That is, in addition to the above few lines, I'd go a step further and make it so we can take a `Duration` object that you've defined -- where we can specify how many weeks, days, hours, etc go into this time interval -- then call a method on it to get something like "P1DT10M10S".